daemon/cmd/daemon_main.go

```diff
@@ -352,6 +352,9 @@ func initializeFlags() {
 	flags.Bool(option.EnableWireguardUserspaceFallback, false, "Enables the fallback to the wireguard userspace implementation")
 	option.BindEnv(Vp, option.EnableWireguardUserspaceFallback)
 
+	flags.String(option.StrictModeCIDR, "", "Enable encryption strict mode to block all unencrypted traffic to specified CIDR")
```

Comment:
Maybe we should name this `encryption-strict-mode-cidr`? "Strict mode" by itself doesn't make it clear that this is about encryption, and not e.g. about policy enforcement.